src/components/accessories/patientOPD/patientOPDTable/PatientOPDTable.tsx

```diff
@@ -57,8 +81,11 @@ const PatientOPDTable: FunctionComponent<IOwnProps> = ({
     return results;
   };
 
-  const onDelete = () => {
-    console.log("delete");
+  const onDelete = (row: OpdDTO) => {
+    if (row.code) {
+      setDeletedObjCode(row.code + "");
+      dispatch(deleteOpd(row.code));
+    }
```

Comment:
I would show an error if something went wrong

Reply:
OK. I have removed the if statement to handle undefined code directly in action creator.